gitenberg/push.py

```diff
@@ -89,7 +89,7 @@ def create_api_handler(self):
 
     def format_desc(self):
         if hasattr(self.book, 'meta'):
-            title = self.book.meta.title
+            title = self.book.meta.title.replace('\n', ' ')
```

Comment:
I'll look at this, but I'm worried that cleaning the title should be done upstream

Reply:
I look at it as a GitHub limitation, rather than a problem with the title (GitHub doesn't allow "control characters" (that's what the error message said) in descriptions). A newline in the repository description doesn't really make sense anyways, but a newline in the book title seems reasonable to me.

Reply:
this is fine, never mind. I thought I tested this, but maybe not.